src/client/java/teammates/client/scripts/DataMigrationHelper.java

```diff
@@ -0,0 +1,34 @@
+package teammates.client.scripts;
+
+public final class DataMigrationHelper {
```

Comment:
Is this going to be useful for all future data migrations? If not, should not be named as such.